README.md

```diff
@@ -1,77 +1,60 @@
+The Spatial Continuity Camera app functions as a hand-held magnifying glass.
+After connecting, it streams the camera feed from the iPhone to the Spatial Continuity Camera app running on Vision Pro.
 
-### SwiftLint
+The Spatial Continuity Camera app can open screenshots from the camera livestream in a separate window for further inspection. 
+If an OpenAI API key is provided, spoken image descriptions will be generated as well. 
 
-The Swiftlint workflow is triggered by every pull request (PR) and checks if the files found in the diff contain any [SwiftLint](https://github.com/realm/SwiftLint) violations.
-You can change the SwiftLint configuration in the `.swiftlint.yml` file found at the root of this repository.
 
-## Continous Delivery Setup
+## Setup and Usage
 
-It is a prerequisite to have access to an Apple Developer Account that allows [TestFlight](https://developer.apple.com/testflight/) releases and create an app in [App Store Connect](https://appstoreconnect.apple.com) that matches the bundle identifier you have defined in the App project.
+1. Upon opening the Spatial Continuity app for the first time, you will be presented with the [SpeziLLM](https://github.com/StanfordSpezi/SpeziLLM) onboarding window where you may enter your OpenAI API key. An OpenAI API key is required for generating spoken image descriptions.
+2. Ensure that both your iPhone and Vision Pro are on the same network. When opening the Spatial Continuity Camera app on iPhone, the connection should be established automatically.
 
-### App Store Connect Access
 
-The [TestFlight](https://developer.apple.com/testflight/) deployment requires access to the App Store Connect API using an API key. Please follow the Apple instructions to [Creating API Keys for the App Store Connect API](https://developer.apple.com/documentation/appstoreconnectapi/creating_api_keys_for_app_store_connect_api). The key needs the `App Manager` access role.
-Store the following information in the following GitHub secrets:
-- `APPLE_ID`: The Apple ID you use to access the App Store Connect API.
-- `APP_STORE_CONNECT_ISSUER_ID`: The issuer ID of the App Store Connect API is displayed in the App Store Connect API keys section.
-- `APP_STORE_CONNECT_API_KEY_ID`: The key ID of the API key created in the App Store Connect API keys section.
-- `APP_STORE_CONNECT_API_KEY_BASE64`: The content of the key created in App Store Connect condensed into a Base64 representation, e.g., using `base64 -i AuthKey_ABCDEFGHIJ.p8 | pbcopy`.
+## Known Issues
 
-### Apple Xcode Certificate and Provisioning Profile
+* Connecting the Spatial Continuity and Spatial Continuity Camera apps may be unreliable at times. Suggestions and PRs welcome! 
```

Comment:
I would rather document this as a GitHub issue; not a great place in the README to have this as a living document.